packages/expo-notifications/src/NotificationCategoriesModule.ts

```diff
@@ -0,0 +1,19 @@
+import { UnavailabilityError } from 'expo-modules-core';
+
+import { NotificationCategoriesModule } from './NotificationCategoriesModule.types';
+
+const notificationCategoriesModule: NotificationCategoriesModule = {
+  async getNotificationCategoriesAsync() {
+    return [];
+  },
+  async setNotificationCategoryAsync() {
+    throw new UnavailabilityError('Notifications', 'setNotificationCategoryAsync');
```

Comment:
I think this is okay, but it is technically a somewhat change for web (throws vs returns null). I think it's the correct choice though versus changing the return type to be nullable.